package.json

```diff
@@ -154,13 +154,13 @@
     "enzyme-to-json": "^3.3.5",
     "eslint": "^5.8.0",
     "eslint-config-airbnb": "^17.0.0",
-    "eslint-config-prettier": "^5.0.0",
-    "eslint-plugin-import": "^2.17.3",
+    "eslint-config-prettier": "^5.1.0",
```

Comment:
I noticed there is a higher version of this package on npm: https://www.npmjs.com/package/eslint-config-prettier

I think a new version was published a few hours after your commit